.github/workflows/tests.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         uses: golangci/golangci-lint-action@v6
         with:
           version: v1.59.1
+          args: --timeout 3m
```

Comment:
Since the config is moved from the root, we must pass the config location so golangci-lint runs correctly.